package.json

```diff
@@ -34,8 +34,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration": "npm run stop && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --runInBand --colors ./src/integration/pages/**/*) ; npm run stop",
-    "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*",
+    "test:integration": "node src/integration/utils/runTests/index.js",
+    "test:integration:ci": "JEST_SILENT_REPORTER_DOTS=true npm run test:integration -- --ci --reporters=jest-silent-reporter",
```

Comment:
Can this env variable `JEST_SILENT_REPORTER_DOTS=true` and this `reporters=jest-silent-reporter` flag also be checked within the `runTests/index.js` as you do with `ci`?

Reply:
I suppose it could be but I'm not sure I see the benefit of doing that as it would add more logic to `runTests/index.js`